roles/powerdns/templates/mariadb.yaml.j2

```diff
@@ -0,0 +1,119 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/name: mariadb
+    app.kubernetes.io/part-of: powerdns
+    app.kubernetes.io/component: mariadb
+  name: mariadb
+spec:
+  podManagementPolicy: OrderedReady
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: mariadb
+      app.kubernetes.io/part-of: powerdns
+      app.kubernetes.io/component: mariadb
+  serviceName: mariadb
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: mariadb
+        app.kubernetes.io/part-of: powerdns
+        app.kubernetes.io/component: mariadb
+    spec:
+      containers:
+        - name: mariadb
+          image: mariadb:11
```

Comment:
Wish we could just use sqlite as a backend but it requires some manual schema generation before the start of the application, which I think is hard to maintain: https://doc.powerdns.com/authoritative/backends/generic-sqlite3.html#setting-up-the-database

Reply:
Using the current container image it does not work because the binary is built without support for sqlite:

```
Unable to load module '/usr/lib64/pdns/libgsqlite3backend.so': /usr/lib64/pdns/libgsqlite3backend.so: cannot open shared object file: No such file or directory
```

It also looks like there are no maintained containers, which support it. So, probably better to just use mariadb here...

Reply:
How many memory is consumed by MariaDB?

Reply:
Around ~200MB:

```
❯ k top pod
NAME                        CPU(cores)   MEMORY(bytes)   
mariadb-0                   1m           217Mi           
powerdns-7dd79b9dfd-mxzps   1m           56Mi 
```

Reply:
@majst01 showed me the official images on docker hub. :laughing: 
It now works with sqlite.